metakb/query.py

```diff
@@ -775,20 +775,23 @@ def search_by_id(self, node_id):
         :param str node_id: node_id
         :return: A dictionary containing the node content
         """
-        node_id = node_id.strip()
-        if '%' not in node_id:
-            concept_name = quote(node_id.split(":", 1)[1])
-            node_id = node_id.split(":", 1)[0] + ":" + concept_name
+        valid_node_id = None
         response = {
-            'query': None,
+            'query': node_id,
             'warnings': []
         }
 
         if not node_id:
             response['warnings'].append("No parameters were entered.")
+        elif node_id.strip() == '':
+            response['warnings'].append("Cannot enter empty string.")
         else:
-            valid_node_id = None
-            response['query'] = node_id
+            node_id = node_id.strip()
```

Comment:
maybe add another `node_id = node_id.replace(" ", "")`? When I was testing some queries, I always accidentally typed a space right after `":"`

Reply:
Hmmm... That's more user error. I don't think we should do this since MOA could have an ID that is a label with spaces in it.